src/scripts/experience/experience.js

```diff
@@ -149,8 +149,14 @@ define([
      */
     Experience.prototype._openIssue = function (issue) {
 
+        var documentBody = $('body');
         var issueData = issue.data;
 
+        // add the modal-issue-active class to the body
+        // this is used in the CSS to bump up the z-index of
+        // some of the sharing icons.
+        documentBody.addClass('issue-modal-active');
```

Comment:
Adding/removing classes on body can be expensive for layout calculations, could the z-index issues be avoided either without this, or by toggling the class on a smaller, more contained element in the DOM?